acceptance/suites/pre_suite/puppet4_compat/70_install_puppet.rb

```diff
@@ -0,0 +1,10 @@
+require 'puppetserver/acceptance/compat_utils'
+
+step "Install Legacy Puppet Agents."
+
+puppet_version = ENV['PUPPET_LEGACY_VERSION']
+if not puppet_version
+  fail "PUPPET_LEGACY_VERSION is not set, e.g. '1.9.3'"
+end
+
+install_puppet_agent_on(nonmaster_agents, {:puppet_agent_version => puppet_agent_version})
```

Comment:
when I get to here I get

```
  Test Case acceptance/suites/pre_suite/puppet4_compat/70_install_puppet.rb reported: #<NameError: undefined local variable or method `puppet_agent_version' for #<Beaker::TestCase:0x007fad4045c870>
```

I think it needs to be `puppet_version` or the variable at line 5 needs to be renamed to `puppet_agent_version`.